sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -628,6 +630,12 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    *   spark.read().text("/path/to/spark/README.md")
    * }}}
    *
+   * You can set the following text-specific options to deal with text files:
+   * <ul>
+   * <li>`lineSep` (default is `\r\n` or `\n`): defines the line separator that should
```

Comment:
I was thinking in that way. However, this appears a default behaviour from Hadoop's `LineRecordReader`. I am worried of a behaviour change (e.g., I guess Windows users get affected by this) .